hmac-mapper/src/main/java/de/intension/validation/CustomClientValidationProvider.java

```diff
@@ -1,43 +1,56 @@
 package de.intension.validation;
 
-import de.intension.protocol.oidc.mappers.HmacExtPairwiseSubMapper;
+import static org.keycloak.models.utils.ModelToRepresentation.toRepresentation;
+import static org.keycloak.protocol.oidc.utils.PairwiseSubMapperValidator.PAIRWISE_FAILED_TO_GET_REDIRECT_URIS;
+import static org.keycloak.protocol.oidc.utils.PairwiseSubMapperValidator.PAIRWISE_REDIRECT_URIS_MISMATCH;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
 import org.keycloak.models.ClientModel;
 import org.keycloak.protocol.ProtocolMapperConfigException;
 import org.keycloak.protocol.oidc.mappers.PairwiseSubMapperHelper;
 import org.keycloak.protocol.oidc.utils.PairwiseSubMapperUtils;
 import org.keycloak.representations.idm.ProtocolMapperRepresentation;
-import org.keycloak.validation.*;
+import org.keycloak.validation.DefaultClientValidationProvider;
+import org.keycloak.validation.ValidationContext;
+import org.keycloak.validation.ValidationError;
+import org.keycloak.validation.ValidationResult;
 
-import java.util.*;
+import de.intension.protocol.oidc.mappers.HmacPairwiseSubMapper;
 
-import static org.keycloak.models.utils.ModelToRepresentation.toRepresentation;
-import static org.keycloak.protocol.oidc.utils.PairwiseSubMapperValidator.PAIRWISE_REDIRECT_URIS_MISMATCH;
-import static org.keycloak.protocol.oidc.utils.PairwiseSubMapperValidator.PAIRWISE_FAILED_TO_GET_REDIRECT_URIS;
+/**
+ * This class is to filter the validation errors pairwise redirect uris mismatch and pairwise failed
+ * to get redirect uris.
+ */
+public class CustomClientValidationProvider extends DefaultClientValidationProvider
+{
 
-public class CustomClientValidationProvider extends DefaultClientValidationProvider {
+    private static final String         PAIRWISE_FIELD_ID         = "pairWise";
+    private static final List<String>   REDIRECT_ERRORS_TO_IGNORE = new LinkedList<>(Arrays.asList(PAIRWISE_REDIRECT_URIS_MISMATCH,
```

Comment:
Due to the fact that mapper doesn't extend `AbstractPairwiseSubMapper` anymore, these errors should never be raised. Pls check